ProjectOxfordFace.podspec

```diff
@@ -11,7 +11,7 @@ Pod::Spec.new do |s|
   s.license          = 'MIT'
   s.author           = { "Project Oxford SDK" => "oxfordGithub@microsoft.com" }
   s.source           = { :git => "https://github.com/Microsoft/Cognitive-Face-iOS.git", :branch => "master" }
-  s.platform     = :ios, '9.0'
+  s.platform     = :ios, '8.0'
```

Comment:
yes, we suggest that this SDK should support from iOS 8.0 since there are still many iOS devices running iOS 8.x